support-files/mysql.spec.sh

```diff
@@ -258,17 +192,18 @@ Name:           MySQL%{product_suffix}
 Summary:        MySQL: a very fast and reliable SQL database server
 Group:          Applications/Databases
 Version:        @MYSQL_RPM_VERSION@
-Release:        %{release}%{?distro_releasetag:.%{distro_releasetag}}
-Distribution:   %{distro_description}
+Release:        %{release}%{dist}
+# Distribution:   %{distro_description}
 License:        Copyright (c) 2000, @MYSQL_COPYRIGHT_YEAR@, %{mysql_vendor}. All rights reserved. Under %{license_type} license as shown in the Description field.
-Source:         http://www.mysql.com/Downloads/MySQL-@MYSQL_BASE_VERSION@/%{src_dir}.tar.gz
+Source:         %{src_dir}.tar.gz
 URL:            http://www.mysql.com/
-Packager:       MySQL Release Engineering <mysql-build@oss.oracle.com> 
+Packager:       Codership Oy <info@galeracluster.com>
 Vendor:         %{mysql_vendor}
-BuildRequires:  %{distro_buildreq}
+# BuildRequires:  %{distro_buildreq}
+#wsrep_patch_tag
 
 # Regression tests may take a long time, override the default to skip them 
-%{!?runselftest:%global runselftest 1}
+%{!?runselftest:%global runselftest 0}
```

Comment:
The default "1" from Oracle was intentional, so that packagers run the tests.
A default "0" makes sense for OBS builds.
I would prefer the "1" if we could specifically override it in OBS, but haven't yet searched for a way to do that.